main.go

```diff
@@ -53,13 +53,17 @@ func getCgroupPath() (string, error) {
 	enabled, err := v2.Enabled()
```

Comment:
You need to check if v2 is enabled first to determine raising error or not, and check if v1 is enabled to determine the path. Otherwise in current change, it returns "/sys/fs/cgroup" when both v1 and v2 enabled.